src/Grids/spectralelement.jl

```diff
@@ -140,7 +141,8 @@ Note: This is accurate only for cubed-spheres of the [`Meshes.EquiangularCubedSp
 @memoize WeakValueDict function SpectralElementGrid2D(
     topology,
     quadrature_style;
-    enable_bubble = false,
+    bubble = enable_bubble,
```

Comment:
Will this result in a breaking release, then?

Reply:
I kept the old one for backward compatibility.